modules/dynamic-apiserver/pkg/cli/install_operation.go

```diff
@@ -257,6 +265,27 @@ func runInstall(k8sclient kubernetes.Interface) func(command *cobra.Command, arg
 				return errors.Wrap(err, "failed to cache tls.key")
 			}
 		}
+		var customDockerRegistry bool
+		var dockerRegistryUrl string
+		var dockerRegistryUsername string
+		var dockerRegistryPassword string
+		if customDockerRegistry, err = getBoolPrompt(command, false, installUseCustomDockerRegistryParam); err != nil {
+			return errors.Wrapf(err, "error processing %s flag", installUseCustomDockerRegistryParam)
```

Comment:
This is not covered by tests, could you add simple tests to cover it?

Reply:
Done